pkg/controller/servicebinding/reconciler_test.go

```diff
@@ -480,7 +501,9 @@ func TestReconcilerReconcileWithConflictingAppSelc(t *testing.T) {
 		requireConditionPresentAndTrue(t, InjectionReady, sbrOutput.Status.Conditions)
 		requireConditionPresentAndTrue(t, BindingReady, sbrOutput.Status.Conditions)
 
-		require.Equal(t, reconcilerName, sbrOutput.Status.Secret)
+		bindingSecret, err := fakeDynClient.Resource(secretsGVR).Get(sbrOutput.Status.Secret, metav1.GetOptions{})
+		require.NoError(t, err)
+		require.Equal(t, sbrOutput.Status.Secret, bindingSecret.GetName())
```

Comment:
`bindingSecret.GetName()` will always be equal to `sbrOutput.Status.Secret` if no error is raised, so this test is irrelevant.